### DISCUSSION TITLE
9G舵机怎么连线？

### DISCUSSION THREAD

**Discussion by wenFrankieeee** [peng-zhihui/ElectronBot#154] [General]
问个问题，谢谢大家： 单独那个9G舵机 内部结构与其余4.3G不同（无电位器，只有电调板），不知是如何改线的？

**Comment by peng-zhihui** [comment]
一样的，都有电位器

**Comment by w964522982** [comment]
你用的应该是360度的9G舵机，他没有电位器，换用180度的才能改线
[![L1I874.jpg](https://s1.ax1x.com/2022/04/14/L1I874.jpg)](https://imgtu.com/i/L1I874)

**Reply by wenFrankieeee** [reply]
谢谢，没留心，随手拿了个之前的9G，拆开一看360的，一看电路又不对，以为哪里出问题了。
